live/eks/terragrunt.hcl

```diff
@@ -0,0 +1,25 @@
+terraform {
+  source = "git::https://github.com/softwaremill/terraform-eks-bootstrap//"
+
+}
+
+include "root" {
+  path = find_in_parent_folders()
+}
+
+inputs = {
+
+  org         = "SML"
+  environment = "test"
+
+  eks_cluster_node_groups = {
+    additional = {
```

Comment:
I don't like this name. "additional" suggest there is some other node pool, but I think it's a only one.
"default"?